docs/content/commands/npx.md

```diff
@@ -6,23 +6,8 @@ description: Run a command from a local or remote npm package
 
 ### Synopsis
 
-```bash
-npm exec -- <pkg>[@<version>] [args...]
-npm exec --package=<pkg>[@<version>] -- <cmd> [args...]
-npm exec -c '<cmd> [args...]'
-npm exec --package=foo -c '<cmd> [args...]'
-
-npx <pkg>[@<specifier>] [args...]
-npx -p <pkg>[@<specifier>] <cmd> [args...]
-npx -c '<cmd> [args...]'
-npx -p <pkg>[@<specifier>] -c '<cmd> [args...]'
-
-alias: npm x, npx
-
---package=<pkg> (may be specified multiple times)
--p is a shorthand for --package only when using npx executable
--c <cmd> --call=<cmd> (may not be mixed with positional arguments)
-```
+<!-- AUTOGENERATED USAGE DESCRIPTIONS START -->
```

Comment:
This one looks like a special case.  We'll likely have to generate the `npm exec` usage here with the prefix `npx` instead.   Fortunately this is possible for the `.md` file, it's not during the runtime of `npx -h` itself because the current setup overwrites process.argv and we never really know how we were called, this is why `npx -h` shows `npm exec` instead for now.